blocks/api/test/serializer.js

```diff
@@ -95,6 +96,10 @@ describe( 'block serializer', () => {
 						content: rawContent,
 					};
 				},
+				encodeAttributes( attributes ) {
+					const { align } = attributes;
+					return { align };
+				},
```

Comment:
>don't you think the `metadata( 'align' )` is better for consistency? It's just another way of parsing the attribute.

I suppose it depends where we take attributes initialization. I tried to communicate my current state of mind earlier at https://github.com/WordPress/gutenberg/issues/391#issuecomment-305581285 . If matching included encoded attributes, I'd agree we wouldn't need this function at all. I might try to spend some time tomorrow exploring AST selectors like the ESLint ones I mentioned in that comment. If those work out, I don't think we'd really want to use them for comment sourcing.